determine_test_coverage.py

```diff
@@ -16,7 +16,9 @@
 run_iter = int(sys.argv[1])
 
 os.system(
-    "docker run -v `pwd`:/ivy -v `pwd`/.hypothesis:/.hypothesis unifyai/ivy:latest python3 -m pytest --disable-pytest-warnings ivy_tests/test_ivy --my_test_dump true > test_names"  # noqa
+    "docker run -v `pwd`:/ivy -v `pwd`/.hypothesis:/.hypothesis unifyai/ivy:latest \
+    python3 -m pytest --disable-pytest-warnings ivy_tests/test_ivy --my_test_dump \
+    true > test_names"
```

Comment:
As these are system commands they shouldn't be tampered with, please revert that change